Assets/SEE/UI/SettingsMenu.cs

```diff
@@ -49,6 +51,31 @@ public class SettingsMenu : PlatformDependentComponent
         /// </summary>
         protected override void StartDesktop()
         {
+            string path = Application.dataPath + "/StreamingAssets/KeyBindings.json";
```

Comment:
The same `Application.dataPath + "/StreamingAssets/KeyBindings.json"` is used above. This redundancy must be avoided. Use a function instead at both occurrences.